kroxylicious-operator/src/test/resources/DerivedResourcesTest/filter-api-not-declared/in-KafkaProxy.yaml

```diff
@@ -9,6 +9,7 @@ kind: KafkaProxy
 apiVersion: kroxylicious.io/v1alpha1
 metadata:
   name: example
+  namespace: my-proxy
```

Comment:
`my-proxy` sounds like it's the name a proxy, not a namespace. I wonder if we should change this to something like `proxy-ns` before we copy+paste it any further.